src/stockwatch/use_cases/importing.py

```diff
@@ -72,13 +72,30 @@ def _process_sell_transaction_row(
     isin: IsinStr,
     row: dict[str, str],
     exchanges: list[CurrencyExchange],
+    cash_settlements: list[CashSettlement],
 ) -> ShareTransaction:
     descr = row["Omschrijving"].split()
     key_index = descr.index("Verkoop")
     nr_stocks = float(descr[key_index + 1].replace(",", "."))
     curr = row["Mutatie"]
     price = Amount(float(descr[key_index + 3].replace(",", ".")), curr)
-    amount = apply_exchange(transaction_datetime, nr_stocks * price, exchanges)
+    amount = Amount(0.0)
```

Comment:
Are the `nr_stocks`, `curr` and `price` filled in during a delisting transaction? If not we might move reading them to the `if not` case, and explicitly setting them to zero in the delisting case.

Especially the `price` might need some extra care? If we could fill in a number there it might make sense (for example calculate it from `nr_stocks` and the `amount` in case of a delisting.

Reply:
Okay, I saw your comment regarding the `price` of a share with zero stocks, and it does make sense. Still the comment above holds I think, just to decrease the complexity of the method we could just enforce `nr_stocks` and `price` to zero in the delisting case and avoid reading them from the csv file.

Reply:
You are right, I now have added the calculation of the price.